pom.xml

```diff
@@ -6,7 +6,15 @@
   <groupId>com.azure</groupId>
   <artifactId>azure-sdk-all</artifactId>
   <packaging>pom</packaging>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-sdk-all;current} -->
+  <version>1.0.0</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.4.0-beta.2</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies> <!-- {x-version-update;com.azure:azure-sdk-all;current} -->
```

Comment:
@sima-zhu I don't believe this is the correct place to have this dependency (which, btw, is declared incorrectly anyways, the version tag belongs on the version line). This is azure-sdk-all which includes client, data and management in its module list. This change needs to be more limited, possibly just in pom.client.xml

Reply:
Not intended to put here.